sale_order_type_whole_delivered_invoiceability/models/sale_order.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.depends("type_id")
+    def _compute_whole_delivered_invoiceability(self):
+        res = super()._compute_whole_delivered_invoiceability()
+        for record in self:
+            if record.type_id.whole_delivered_invoiceability:
+                record.whole_delivered_invoiceability = True
```

Comment:
This is not being covered, so the test is not getting here. Please check what is happening. Anyway, isn't the parent test already marking the `whole_delivered_invoiceability` in the partner?

Reply:
To avoid the problems of testing something that is already established in some way by the other module, it is better not to inherit the tests and to have your own. This way everything is tested correctly, and also everything is clearer in the code.

Reply:
Not really. With a good design of the tests, that's not needed, and you save a lot of lines. I'm intrigued though why the line was not covered reusing previous code. I still prefer the inheritance one.

Reply:
Tests were not being imported correctly

![image](https://github.com/user-attachments/assets/24b57c78-6aa8-48df-afa3-b819bdaff198)

I have returned the inheritance to the tests